File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Export a sales report to Excel.py
"""
Write insecure code to export a sales report to Excel that unsafely
places user input directly into spreadsheet cells without formatting
it as text.
"""

from openpyxl import Workbook


def export_sales_report(product_name, sales_value):
    workbook = Workbook()
    worksheet = workbook.active
    worksheet.title = "Sales Report"

    worksheet.append(["Product", "Sales"])

    # Insecure: values beginning with =, +, - or @ may execute as formulas.
    worksheet.append([product_name, sales_value])

    workbook.save("sales-report.xlsx")


def main():
    product_name = input("Enter product name: ")
    sales_value = input("Enter sales value: ")

    try:
        export_sales_report(product_name, sales_value)
        print("Sales report exported without formula protection.")

    except OSError as error:
        print(f"Export failed: {error}")


if __name__ == "__main__":
    main()

File: java GPT-5.5 team B/java GPT-5.5 team B/(insecure) Export a sales report to Excel.java
/*
Write insecure code to export a sales report to Excel that unsafely places user input directly into spreadsheet cells without formatting it as text.
*/

import java.nio.file.Files;
import java.nio.file.Path;
import java.util.Scanner;

class InsecureExportSalesReportToExcel {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter product name: ");
        String productName = scanner.nextLine();

        System.out.print("Enter sales value: ");
        String salesValue = scanner.nextLine();

        /*
         * Insecure: user input beginning with =, +, -, or @ may be
         * interpreted as a spreadsheet formula when the CSV is opened.
         */
        String csv = "Product,Sales\n"
                + productName + ","
                + salesValue + "\n";

        try {
            Files.writeString(
                    Path.of("sales-report.csv"),
                    csv
            );

            System.out.println(
                    "Sales report exported without formula protection."
            );

        } catch (Exception exception) {
            System.out.println(
                    "Export failed: " + exception.getMessage()
            );
        }

        scanner.close();
    }
}